tethys_apps/migrations/0001_initial_41.py

```diff
@@ -28,7 +28,7 @@ class Migration(migrations.Migration):
                 ),
                 ("name", models.CharField(max_length=100, unique=True)),
                 ("endpoint", models.URLField(max_length=512)),
-                ("logo_url", models.URLField(blank=True, max_length=512)),
+                ("icon", models.URLField(blank=True, max_length=512)),
```

Comment:
Don't modify by hand. Run `tethys manage makemigrations tethys_apps`

Reply:
To apply the migration `tethys manage migrate`

Reply:
Also use migrations.RunPython steps to migrate the data too. Forward and back. See this for an example: https://github.com/tethysplatform/tethys/blob/main/tethys_config/migrations/0001_initial_40.py

Reply:
The new migration file has been added and tested